LibraryPolicyCheck.py

```diff
@@ -280,6 +281,10 @@
     "libz.so.1",
 )
 
+_policy_name_misdetects_re = re.compile(
+    '^libreoffice(-.*)?|^librecad|^libressl'
```

Comment:
The ^'s in all of the alternatives are redundant. Please remove. Instead what is missing I think is the end anchor '$'.